AgoraSocialApp/SvelteKit/src/components/Home.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import { conversations } from '$stores/conversations';
+  import { user } from '$stores/user';
+  import { token } from '$stores/session';
+  import Logo from './UI/Logo.svelte';
+  import { AuthService } from '$components/Auth/auth.service';
+
+  const authService = AuthService.getInstance();
+	const forecast = async () => {
+    const headers = new Headers();
+    headers.append('Authorization', `Bearer ${$token}`);
+		const res = await fetch('/WeatherForecast', { headers });
```

Comment:
Is there a way to intercept a fetch, like an angular interceptor, so we can append the bearer in one place instead of all places?

Reply:
yeah and i got the token subscribing as well